demos/classification_demo/cpp/main.cpp

```diff
@@ -328,13 +325,23 @@ int main(int argc, char *argv[]) {
         }
 
         //// --------------------------- Report metrics -------------------------------------------------------
-        slog::info << slog::endl << "Metric reports:" << slog::endl;
+        slog::info << "Metric reports:" << slog::endl;
         metrics.printTotal();
+        slog::info << "Avg time:" << slog::endl;
```

Comment:
```suggestion
```
It should be obvious. Also, `PerformanceMetrics` implementation could be changed.

Reply:
Removed everywhere.